pysidt/sidt.py

```diff
@@ -56,10 +56,10 @@ def __init__(
         iter_max=2,
         iter_item_cap=100,
         r=None,
-        r_bonds=[1, 2, 3, 1.5, 4],
```

Comment:
In general it's possible to do this change, but I believe this would require a more complicated refactor because of the manipulations we do in  get_extensions.

Reply:
Fixed!